python/nx-cugraph/_nx_cugraph/core.py

```diff
@@ -45,18 +45,27 @@ def update_text(text, lines_to_add, target, indent=" " * 8):
     return f"{text[:start]}{begin}{to_add}\n{indent}{text[stop:]}"
 
 
+def dq_repr(s):
+    """Return repr(s) quoted with the double quote preference used by black."""
+    rs = repr(s)
+    if rs.startswith("'") and '"' not in rs:
+        rs = rs.strip("'").replace('"', '\\"')
```

Comment:
Why do we need `.replace('"', '\\"')`? We skip this if `'"' not in rs`
```suggestion
        rs = rs.strip("'")
```